src/app/destinyTrackerApi/reviewsFetcher.ts

```diff
@@ -1,12 +1,72 @@
 import { ItemTransformer } from './itemTransformer';
 import { PerkRater } from './perkRater';
 import { UserFilter } from './userFilter';
-import { D1ItemReviewResponse, D1CachedItem } from '../item-review/destiny-tracker.service';
 import { ReviewDataCache } from './reviewDataCache';
 import { handleErrors } from './trackerErrorHandler';
 import { loadingTracker } from '../ngimport-more';
 import { D1Item } from '../inventory/item-types';
 import { dtrFetch } from './dtr-service-helper';
+import { D1ItemReviewResponse, D1ItemUserReview } from '../item-review/d1-dtr-api-types';
+import { DtrReviewer } from '../item-review/dtr-api-types';
+
+/** A single user's review for a D1 weapon. */
+export interface ActualD1ItemUserReview {
+  /**
+   * Is this reviewer a featured reviewer?
+   * Broken in D2.
+   */
+  isHighlighted: boolean;
```

Comment:
Since this is a D1 specific type, does it matter if it's broken in D2?

Reply:
Kind of? We receive it in the API, but the value is currently not being set, but we calculate and replace it internally on receipt and use it.